lib/core/src/sdk.rs

```diff
@@ -172,14 +168,31 @@ impl LiquidSdk {
         tokio::spawn(async move {
             let mut shutdown_receiver = cloned.shutdown_receiver.clone();
             let mut updates_stream = cloned.status_stream.subscribe_swap_updates();
+            let send_swap_state_handler = SendSwapStateHandler::new(
```

Comment:
Any reason the SendSwapStateHandler is not stored in the LiquidSdk struct?

Reply:
In my mind it is not a statefull struct it is just a stateless handler that can be instantiate on demand leaving the LiquodSdk struct simpler.
I assume saving it on the LiquidSdk struct will make sense if it is abstracted behind a trait but this is not the case and I don't think we need it atm.
Does it make sense to you as well?

Reply:
Yeah, I thought about that because it doesn't need the trait. I'm happy